README.md

```diff
@@ -1,2 +1,133 @@
-# sdm-versions-manager
-Creation of a SDM database of data models’ versions with information of the current version of the data model and the corresponding hash code in GitHub repository.
+# SDM Versions Manager
+
+## Overview
+
+**sdm-versions-manager** is a Python-based service designed to create a database of Smart Data Models (SDMs) versions.
+
+ It retrieves information about the current version of each data model and its corresponding hash code from a GitHub repository. 
+ 
+ This service aims to facilitate the management and tracking of data model versions, ensuring that users have access to the most up-to-date information.
+
+## Features
+
+- Fetches historical commit data from GitHub for specified data models.
+- Extracts version information and commit hashes from the schema files.
+- Populates a MongoDB database with the current versions of data models.
```

Comment:
Since the historic versions fetch happens only once in the "initial_population.py" and that fills the database with historic versions. The service purpose is to always keep the database up to date with current versions. So that the database and Github are always synchronized. => This is what I tried to explain in the last point. 
But I can reformulate it.